cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -329,6 +367,75 @@ def test_ssa_template(local_setup_provider, provider, soft_assert, vm_analysis_p
         soft_assert(c_fs_drivers != '0', "fs drivers: '{}' != '0'".format(c_fs_drivers))
 
 
+@pytest.mark.tier(2)
+@pytest.mark.long_running
+def test_ssa_schedule(ssa_vm, schedule_ssa, soft_assert, appliance):
+    """ Tests SSA can be performed and returns sane results
+
+    Metadata:
+        test_flag: vm_analysis
+    """
+    e_users = None
+    e_groups = None
+    e_packages = None
+    e_services = None
```

Comment:
**QUESTION**
Why are these None assignments needed?

Reply:
Initially, all entities are not available. After SSA fields are compared with above entities.

Reply:
They are not needed, you make assignments below in `if ssa_vm.system_type != WINDOWS:` block

Reply:
If Linux VM/Instance then it will take as it is, else it will check for Windows and will update accordingly for further cimparison